### ISSUE TITLE
Wrong Cursor column numbers

### ISSUE THREAD

**Issue by rexx6** [microsoft/vscode#91990]
- VSCode Version: 1.42.1
- OS Version: windows7

Does this issue occur when all extensions are disabled?: Yes

![image](https://user-images.githubusercontent.com/28583017/75845361-3042c980-5e1c-11ea-9ccb-b0eeb582a191.png)

[https://github.com/microsoft/vscode/issues/91060](Pervious Post)

Sorry for the late response.
When using fixed-width font in Korean, as shown in the figure, the Korean 1-character uses two columns.
Therefore, in the lower figure above, the column should be marked 19 instead of 17.

Here Sample : `12345678901234동해물과`

**Comment by alexdima** [comment]
Please use a different font, like Inconsolata.

**Comment by alexdima** [comment]
This is working as designed. The indicator in the status bar works in the following way:
* for a tab character, it counts the number of spaces the tab was rendered with
* for any other Unicode code point, it counts 1, regardless of the character font render width.

If you will check across editors, you will see that not all editors agree on what is useful to render there. But we have decided to render what I have documented above.
